examples/SuperResolution/odm/preprocessor.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from kaolin.transforms import transforms as tfs
+
+
+class VoxelODMs:
```

Comment:
you can make it inherit from kaolin.Transforms ? (also you can remove `__repr__')

Reply:
Yup!